app/utils/initialize/end-points.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function(app, webSocketUserSettings) {
+  app.post("/settings/onrequestdata", req => {
+    webSocketUserSettings.setOnRequestData(req.body);
+  });
```

Comment:
This is one of the app's main business logic, should definitely not be in the utils folder.

Reply:
Where'd you place it? I'm a bit confused about structuring this project.

Reply:
https://github.com/c-hive/ws-anywhere/pull/2#issuecomment-487567090